src/lib/isLength.js

```diff
@@ -15,5 +15,9 @@ export default function isLength(str, options) {
   const presentationSequences = str.match(/(\uFE0F|\uFE0E)/g) || [];
   const surrogatePairs = str.match(/[\uD800-\uDBFF][\uDC00-\uDFFF]/g) || [];
   const len = str.length - presentationSequences.length - surrogatePairs.length;
-  return len >= min && (typeof max === 'undefined' || len <= max);
+  const isInsideRange = (len >= min && (typeof max === 'undefined' || len <= max));
```

Comment:
suggestion: Could you add line breaks between variable definitions, block statement/conditions, etc? This will add to legibility. I know, this should be covered by automatic formatter and linter steps, which is a different construction site 😄 .

```suggestion
  const isInsideRange = len >= min && (typeof max === 'undefined' || len <= max);

  if (isInsideRange && Array.isArray(options?.discreteLengths)) {
      return options.discreteLengths.some(discreteLen => discreteLen === len);
  }

  return isInsideRange;  
```

Reply:
> suggestion: Could you add line breaks between variable definitions, block statements/conditions, etc? This will add to legibility. I know, this should be covered by automatic formatter and linter steps, which is a different construction site 😄 .

Yess added blank lines in between variable definitions and block statements.